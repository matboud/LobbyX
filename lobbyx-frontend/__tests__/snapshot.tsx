import { render } from "@testing-library/react";
import Home from "@/app/page";
import ReduxProvider from "@/redux/provider";
import { act } from "@testing-library/react";

it("page renders correctly", async () => {
  let container;

  await act(async () => {
    const result = render(
      <ReduxProvider>
        <Home />
      </ReduxProvider>
    );
    container = result.container;
  });

  expect(container).toMatchSnapshot();
});
