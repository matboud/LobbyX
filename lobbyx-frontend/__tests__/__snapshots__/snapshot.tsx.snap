// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page renders correctly 1`] = `
<div>
  <div
    class="w-full flex justify-center items-center"
  >
    Error: 
    fetch is not defined
  </div>
</div>
`;
